cabal-testsuite/PackageTests/Check/InvalidModuleName/cabal.test.hs

```diff
@@ -0,0 +1,13 @@
+import Test.Cabal.Prelude
+
+main = cabalTest $ do
+    r <- fails $ cabal' "check" ["--ignore=unknown-directory"]
+    assertOutputContains "is invalid on Windows" r
+    assertOutputContains "Aux" r
+    assertOutputContains "Exe/Aux" r
+    assertOutputContains "Test/Aux" r
+    assertOutputContains "Bench/Aux" r
+    assertOutputContains "Exe/Aux/Test" r
+    assertOutputContains "Test/Aux/Test" r
+    assertOutputContains "Bench/Aux/Test" r
```

Comment:
Why all these `Contains` when the plain check against `cabal.out` would do?

Reply:
I suppose it is better to make the test explicit about what you wish to see, in case the output changes in future and it's blindly accepted.

Reply:
> I suppose it is better to make the test explicit about what you wish to see, in case the output changes in future and it's blindly accepted.

If you have `Exe/Aux` but not `Aux` by itself,  `assertOutputContains "Aux" r` would blindly accept it. Not so with `cabal.out`.

But the main point is that the golden test output *is* what should remain unchanged over time, so I would ask you to axe this.

Reply:
After thinking about it, I still think `assertOutputContains` adds very little and is a cost to the maintainer.